external/agemo/proto/publisher/v1/publisher.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I think that you should add agemo as a submodule, rather than copying its files.

Reply:
Okay, fair enough. I have updated it to be a submodule